src/main/resources/swagger/api-docs.yaml

```diff
@@ -55,6 +55,44 @@ paths:
         - test
   '/api/clusters':
     get:
+      summary: List all active clusters
+      description: List all active clusters, optionally filtering on a set of labels
+      operationId: listClusters
+      tags:
+        - cluster
+      parameters:
+        - in: query
+          name: _labels
+          description: |
+            Optional label key-value pairs to filter results by. Example: key1=val1,key2=val2.
+
+            Note: this string format is a workaround because Swagger doesn't support free-form
+            query string parameters. The recommended way to use this endpoint is to specify the
+            labels as top-level query string parameters. For instance: GET /api/clusters?key1=val1&key2=val2.
+          required: false
+          type: string
+      responses:
+        '200':
+          description: List of clusters
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Cluster'
+        '400':
+          description: Bad Request
+          schema:
+            $ref: '#/definitions/ErrorReport'
+        '500':
+          description: Internal Error
+          schema:
+            $ref: '#/definitions/ErrorReport'
+      security:
+        - googleoauth:
+            - openid
+            - email
+            - profile
+  '/api/clusters/includeDeleted':
```

Comment:
is this alright or would we prefer `/api/clusters/includeDeleted`

Reply:
I'd prefer `&includeDeleted`. I'm okay if we want to tell Swagger the parameter is mandatory as long as Leo behaves correctly (i.e. in all four cases: no query string, `&includeDeleted`, `&includeDeleted=true`, and `&includeDeleted=false`).

Reply:
+1, I agree with @helgridly